### PR TITLE
Fix App.configs setup

### PR DESCRIPTION
**Pull request by penev92**
Right now only `OpenRA.Test.csproj` has a proper App.config. The other 2 are just files dropped in the main repo dir without the .sln file knowing about them.

### CODE REVIEW

**Comment by Mailaender** [comment]
You will need to `cp` those in the `Makefile` manually.

**Comment by penev92** [comment]
They will also need to be manually removed on `clean`.

**Comment by pchote** [comment]
Can we get this fixed and merged?

**Comment by reaperrr** [comment]
@penev92 Any chance you could fixup this soon-ish?

File: OpenRA.Utility/App.config [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1" appliesTo="v1.0.3705">
-      <dependentAssembly>
-        <assemblyIdentity name="Microsoft.Build.Framework" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-100.0.0.0" newVersion="12.0.0.0" />
-      </dependentAssembly>
-    </assemblyBinding>
-  </runtime>
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <runtime>
```
Comment (pchote) [inline OpenRA.Utility/App.config]: Are the differences between this `.config` and the OpenRA.Game one important?
If not, can we please make them consistent?

**Comment by Phrohdoh** [comment]
Why are these files necessary?

**Comment by Phrohdoh** [comment]
Penev seems to be busy with life for the time being, could someone take this over or close it?

**Comment by pchote** [comment]
@penev92 indicated in IRC that he's not going to finish this, and it doesn't seem to solve any real issues so closing.

**Pull request closed without merging**